cmd/client/start.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+
+	bbgrpc "github.com/jmbarzee/bitbox/grpc"
+)
+
+var cmdStart = &cobra.Command{
+	Use:   "start",
+	Short: "start",
+	Long:  "Start a process on the bitbox server",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			panic(errors.New("Require atleast a single command as an argument"))
+		}
+
+		job := jobStart{
+			command:   args[0],
+			arguments: args[1:],
+		}
```

Comment:
have you tried this with implicitly invoking a shell? 
example `/bin/bash -c "ls -lah ~"`